lib/eos/ecc/point.js

```diff
@@ -1,10 +1,24 @@
-var assert = require('assert')
-var BigInteger = require('bigi/lib/bigi')
+import bigInt from 'big-integer';
+import { assert } from './../helper';
 
-var THREE = BigInteger.valueOf(3)
+const ONE = bigInt.one;
+const ZERO = bigInt.zero;
+const THREE = bigInt(3);
 
-function Point (curve, x, y, z) {
-  assert.notStrictEqual(z, undefined, 'Missing Z coordinate')
+function testBit(bigIntVal, index) {
```

Comment:
This is horrendously inefficient. Can't you use bitmasks for this?

Reply:
Fixed